bundles/Planning/Resources/assets/planning.js

```diff
@@ -144,13 +144,13 @@ require('./planning.css');
             getStatusLevelClass: function (user, sprint) {
                 let remainingEstimateUser = this.getRemainingEstimatUser(user, sprint);
 
-                if (remainingEstimateUser > 35) {
+                if (remainingEstimateUser > 70) {
                     return 'remaining-critical';
-                } else if (remainingEstimateUser > 28 && remainingEstimateUser <= 30) {
+                } else if (remainingEstimateUser === 60) {
```

Comment:
Why this if, where "warning" is 60 precisely?
Below you have "danger" for the range (53,70]...

Reply:
Because 60 precisely is usually only when people have booked vacation for the whole sprint.

Reply:
I will move it to be after the range :)